panc/src/main/scripts/panlint/panlint.py

```diff
@@ -465,7 +466,7 @@ def main():
 
     if not args.paths:
         print 'No files were provided, not doing anything'
-        sys_exit(0)
+        return 0
```

Comment:
IMHO this is a (user) error. Since the user ran the tool they wanted an effect, unless they passed --help or -v or something.